src/uu/cp/src/cp.rs

```diff
@@ -713,7 +721,7 @@ impl Attributes {
             "ownership" => self.ownership = preserve_yes_required,
             "timestamps" => self.timestamps = preserve_yes_required,
             "context" => self.context = preserve_yes_required,
-            "links" => self.links = preserve_yes_required,
+            "link" | "links" => self.links = preserve_yes_required,
```

Comment:
This is probably due to that "feature" of GNU where it accepts unambiguous prefixes of values. `l`, `li`, etc. will also work. But this fix is probably fine for now.